RecoLocalTracker/SubCollectionProducers/src/HITrackClusterRemover.cc

```diff
@@ -318,13 +318,8 @@ void HITrackClusterRemover::process(OmniClusterRef const &ocluster, SiStripDetId
   if (pblocks_[subdet - 1].usesSize_ && (cluster->amplitudes().size() > pblocks_[subdet - 1].maxSize_))
     return;
   if (!fromTrack) {
-    int clusCharge = 0;
-    for (std::vector<uint8_t>::const_iterator iAmp = cluster->amplitudes().begin(); iAmp != cluster->amplitudes().end();
-         ++iAmp) {
-      clusCharge += *iAmp;
-    }
-    if (pblocks_[subdet - 1].cutOnStripCharge_ &&
-        (clusCharge > (pblocks_[subdet - 1].minGoodStripCharge_ * sensorThickness(detid))))
+    if (pblocks_[subdet - 0].cutOnStripCharge_ &&
+        (cluster->charge() > (pblocks_[subdet - 1].minGoodStripCharge_ * sensorThickness(detid))))
```

Comment:
I'm confused why the first conditional is on ```subdet -0```, while the other is on ```subdet - 1```.

Reply:
@pieterdavid 
ping
[in case the last comment got lost]

Reply:
That was an unintentional change indeed - reverted now. Thanks for spotting it!